drivers/Makefile.dep

```diff
@@ -81,16 +81,18 @@ ifneq (,$(filter bm%280,$(USEMODULE)))
   USEMODULE += bmx280
 endif
 
-ifneq (,$(filter cc110x,$(USEMODULE)))
-  USEMODULE += ieee802154
+ifneq (,$(filter cc110%,$(USEMODULE)))
+  USEMODULE += cc110x
+  USEMODULE += cc1xxx_common
   USEMODULE += luid
+  USEMODULE += netif
   USEMODULE += xtimer
-  ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
-    USEMODULE += gnrc_cc110x
-  endif
   FEATURES_REQUIRED += periph_gpio
   FEATURES_REQUIRED += periph_gpio_irq
   FEATURES_REQUIRED += periph_spi
+  ifneq (,$(filter gnrc_ipv6,$(USEMODULE)))
+    USEMODULE += gnrc_sixlowpan_router_default
```

Comment:
Why fix to router? And why pull in every default dependency.

I think what you are trying to do is something like

https://github.com/RIOT-OS/RIOT/blob/c12bd5edb161916bdc4f218d7ef028ad916b9562/Makefile.dep#L167-L183